python/cusignal/filter_design/fir_filter_design.py

```diff
@@ -255,22 +256,22 @@ def firwin(
 
     # Insert 0 and/or 1 at the ends of cutoff so that the length of cutoff
     # is even, and each pair in cutoff corresponds to passband.
-    cutoff = cp.hstack(([0.0] * pass_zero, cutoff, [1.0] * pass_nyquist))
+    cutoff = np.hstack(([0.0] * pass_zero, cutoff, [1.0] * pass_nyquist))
 
     # `bands` is a 2D array; each row gives the left and right edges of
     # a passband.
     bands = cutoff.reshape(-1, 2)
 
     # Build up the coefficients.
     alpha = 0.5 * (numtaps - 1)
```

Comment:
This seemingly replaces filter creation from cupy to numpy, but have we benchmarked E2E performance (e.g. applying that filter to in GPU data). I'm curious if the extra H->D copy negates performance you get there?